webp/.gitignore

```diff
@@ -0,0 +1,2 @@
+libwebp
```

Comment:
yeah, mainly the libwebp directory is downloaded from github inside build.gradle, so we want to avoid it to be accidentally checked in.  If add to the upper layer directory's .gitignore,  I felt that later we will have to move it back whenever someone add a libwebp directory ( mainly I felt webp is going to be popular, more webp to come ).